Packages/com.unity.inputsystem/InputSystem/Editor/InputActionImporter.cs

```diff
@@ -203,10 +213,7 @@ public override void OnImportAsset(AssetImportContext ctx)
                 };
 
                 if (InputActionCodeGenerator.GenerateWrapperCode(wrapperFilePath, maps, asset.controlSchemes, options))
-                {
-                    // Inform database that we modified a source asset *during* import.
-                    AssetDatabase.ImportAsset(wrapperFilePath);
-                }
+                    EditorApplication.update += DelayedRefresh;
```

Comment:
Can we replace this with `EditorApplication.delayCall += AssetDatabase.Refresh;` ?

Reply:
No. The point of the wrapper is to remove itself from `EditorApplication.update` once invoked once.

Reply:
delayCall does that automatically.

Reply:
Oh. I did not see that you replaced `update` with `delayCall`, which I did not know about. Then this is much nicer indeed!